_2019_pages/agenda.html

```diff
@@ -29,10 +29,28 @@ <h1>{{ page.title }}</h1>
         <p>For the practical part, we encourage you to bring your own problem to work on.</p>
         <p>
             Note that we are not planning to give an introduction to Python/NumPy/SciPy, but there
-            are plenty of free online resources to learn the basics before the start of the School
-            (see <a href="https://python.g-node.org/wiki/introductory_material">here</a> for
-            suggestions).
+            are plenty of free online resources to learn the basics before the start of the School:
         </p>
+        <ul>
+            <li>
+                for inexperienced programmers:
+                <ul>
+                    <li><a href="https://link.springer.com/book/10.1007/978-3-319-32428-9">https://link.springer.com/book/10.1007/978-3-319-32428-9</a></li>
+                    <li><a href="https://python.swaroopch.com/">https://python.swaroopch.com/</a></li>
+                    <li><a href="https://docs.python-guide.org/">https://docs.python-guide.org/</a></li>
```

Comment:
I would move this one to 'more experienced programmers'. It seems like a good companion to 'dive into python 3'.